components/admin_console/license_settings/trial_banner/trial_banner.tsx

```diff
@@ -160,22 +159,16 @@ const TrialBanner: React.FC<TrialBannerProps> = ({
                 );
         }
         trialButton = (
-            <button
```

Comment:
I'd rather not change this over to `StartTrialBtn` at the moment. Looks like there's some complex interaction between this component, its useEffect hooks, which will request a license depending on some preference values, [`LicenseSettings`](https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/license_settings/license_settings.tsx#L360) which renders it, and LicenseSettings' [`requestTrial`](https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/license_settings/license_settings.tsx#L207) method which prevents duplicate requests.

I think if we replace this `<button/>` with `<StartTrialBtn/>` now, we'll lose some of those guards against making duplicate requests. Not sure what all the implications are for that.

Reply:
ok, I have to agree because the logic stablished for the team edition scenario is very complex and also because this is a difficult change to validate (really hard to do it locally, and at the time, it is imposible to do it with a spinwick server)... in conclusion, the risk is not worth the benefit. Thanks for pointing this out @neallred